assets/components/featureList/featureList.jsx

```diff
@@ -4,6 +4,8 @@
 
```

Comment:
Same concern as the other component. It looks like you are using a component in a unique place.

Reply:
For now I am, it will be in many pages though =).